.releaserc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    ["@semantic-release/npm", { "npmPublish": true }],
+    "@semantic-release/changelog",
```

Comment:
As far as I understand this generates a `CHANGELOG.md`. Since it is not committed, do we need it?